### PR TITLE
Replace wmi queries with win32 api calls

### PR DESCRIPTION
**Pull request by narph**
Fixes https://github.com/elastic/beats/issues/11840

Alternative to disable config option here https://github.com/elastic/beats/issues/3249

Addresses high CPU load on windows

### CODE REVIEW

File: sigar_windows.go [hunk 1]
```diff
@@ -371,15 +371,33 @@ func (self *ProcArgs) Get(pid int) error {
...
+		return nil
+	}
+	var args []string
+	if argsW, err := windows.ReadProcessUnicodeString(handle, &userProcParams.CommandLine); err == nil {
+		args, err = windows.ByteSliceToStringSlice(argsW)
+		if err != nil {
+			args = nil
+		}
 	}
+	var process = Win32_Process{CommandLine: &args[0]}
```
Comment (andrewkroh) [inline sigar_windows.go]: I think we can remove the `Win32_Process` struct completely.

File: sigar_windows.go [hunk 2]
```diff
@@ -371,15 +371,33 @@ func (self *ProcArgs) Get(pid int) error {
...
+		return errors.Wrapf(err, "NtQueryProcessBasicInformation failed for pid=%v", pid)
+	}
+	if err != nil {
+		return nil
+	}
+	userProcParams, err := windows.GetUserProcessParams(handle, pbi)
+	if err != nil {
+		return nil
+	}
+	var args []string
```
Comment (andrewkroh) [inline sigar_windows.go]: Instead of creating a variable, what about just using `self.List` directly.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Added missing runtime import for FreeBSD. #104
 - Handle nil command line in Windows processes. #110
+- Replaced the WMI queries with win32 apis due to high CPU usage. #11840
```
Comment (ruflin) [inline CHANGELOG.md]: Should this go under `unrealeased`?

Comment (narph) [reply CHANGELOG.md]: all good now

**Pull request merged**